src/Layout/redist/targets/GenerateBundledVersions.targets

```diff
@@ -1318,8 +1318,21 @@ Copyright (c) .NET Foundation. All rights reserved.
                               TargetingPackName="NETStandard.Library.Ref"
                               TargetingPackVersion="$(NETStandardLibraryRefPackageVersion)"
                               />
-
-    <!-- Supported Windows versions -->
+    
+    <!--
+      Supported Windows versions using CsWinRT 3.0 (.NET 10 is the baseline). This leverages the patch version of
+      the Windows SDK version to allow multi-targeting, which is required to allow customers to gradually migrate.
+      Both CsWinRT 3.0 and 2.x will be supported for a period of time, and packages can target both of them.
+    -->
+    <WindowsSdkSupportedTargetPlatformVersion Include="10.0.26100.1" WindowsSdkPackageVersion="$(MicrosoftWindowsSDKNETRef10_0_26100_1PackageVersion)" MinimumNETVersion="10.0" />
```

Comment:
@dsplaisted I imagine we'd need some tweak somewhere in the .NET SDK so that if you're targeting `.1`, `TargetPlatformVersion` and all the other properties specifically for the Windows SDK still end up using `.0`, but I'm not sure where that logic should be. Would you be able to help me with this?

Reply:
So I'm wondering if I should update the actual implementation of `GetTargetPlatformVersion` in MSBuild so that if the version is `10.0.XXXXX.1`, it just sets the build number to `0` instead? Would that be the correct fix? 🤔